### PR TITLE
[#118] [Bug] Text not showing in Popup Dialog 

### LINKED ISSUE #118: [Bug] Text not showing in Popup Dialog
## Issue

When navigate back from SurveyDetail screen, confirmation dialog appears and there is no text showing.

Screen Shot 2022-10-20 at 2 36 27 PM

## Expected

When navigate back from SurveyDetail screen, confirmation dialog with text should appear.

## Steps to reproduce

1. Go to survey question home screen
2. Go to survey question detail and scroll to second page
3. Press back

### PR DESCRIPTION
**Pull request by kaungkhantsoe**
Closes #118 

## What happened 👀

When navigate back from SurveyDetail screen, confirmation dialog appears and there is no text showing.

Screen Shot 2022-10-20 at 2 36 27 PM

## Insight 📝

After attempting to fix the flickering while zoom in and zoom out navigation by wrapping the nav host inside `Surface`, the text doesn't show anymore. It turns out if the view are drawn over the surface, we need to set the color of the text or it will use the device's current theme(in this case `white` color).

## Proof Of Work 📹

https://user-images.githubusercontent.com/32578035/196888974-9434b85f-b7df-42cf-8b93-cd9b92bd4f9a.mp4

### CODE REVIEW

**Comment by ryan-conway** [comment]
@kaungkhantsoe When drawn over a `Surface`, composables will attempt to use theme colours unless explicitly set. We could also handle this issue by removing the `backgroundColor` attribute from the dialogs, resulting in a dark dialog. Is this something we will consider or will we only use light themed components in the app? 😄

**Comment by kaungkhantsoe** [comment]
> @kaungkhantsoe When drawn over a `Surface`, composables will attempt to use theme colours unless explicitly set. We could also handle this issue by removing the `backgroundColor` attribute from the dialogs, resulting in a dark dialog. Is this something we will consider or will we only use light themed components in the app? 😄
> 
> 

@ryan-conway The problem with dark and light mode depends on the device. In `Samsung` it shows correctly as follow:

https://user-images.githubusercontent.com/32578035/197114197-d37f7599-886e-4055-8cdb-e68252ef2460.mp4

In `xiaomi`, the texts don't appear like I said in previous retrospective:

Screenshot_20221021_114742
Screenshot_20221021_114715

File: app/src/main/java/com/kks/nimblesurveyjetpackcompose/ui/presentation/survey/SurveyQuestionScreen.kt [hunk 1]
```diff
@@ -15,7 +15,13 @@ import androidx.compose.ui.tooling.preview.Preview
...
 import androidx.compose.ui.unit.sp
 import com.kks.nimblesurveyjetpackcompose.R
-import com.kks.nimblesurveyjetpackcompose.model.QuestionDisplayType.*
+import com.kks.nimblesurveyjetpackcompose.model.QuestionDisplayType.CHOICE
+import com.kks.nimblesurveyjetpackcompose.model.QuestionDisplayType.DROPDOWN
+import com.kks.nimblesurveyjetpackcompose.model.QuestionDisplayType.NONE
+import com.kks.nimblesurveyjetpackcompose.model.QuestionDisplayType.NPS
+import com.kks.nimblesurveyjetpackcompose.model.QuestionDisplayType.SMILEY
+import com.kks.nimblesurveyjetpackcompose.model.QuestionDisplayType.STARS
+import com.kks.nimblesurveyjetpackcompose.model.QuestionDisplayType.THUMBS
```
Comment (ryan-conway) [inline app/src/main/java/com/kks/nimblesurveyjetpackcompose/ui/presentation/survey/SurveyQuestionScreen.kt]: out of scope change here 🤔

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -46,4 +46,5 @@
     <string name="survey_question_thanks">Thanks for taking\nthe survey.</string>
     <string name="survey_question_not_at_all_likely">Not at all Likely</string>
     <string name="survey_question_extremely_likely">Extremely Likely</string>
+    <string name="survey_question_your_thoughts">Your thoughts</string>
```
Comment (ryan-conway) [inline app/src/main/res/values/strings.xml]: same here 🤔

**Review by ryan-conway — COMMENTED** [review]
`AlertDialogs.kt` LGTM 🙏🏻 

Seems like theres an [extra commit](https://github.com/kaungkhantsoe/NimbleSurveyJetpackCompose/pull/119/commits/31f78320dd6705afb6e8fc6fc96e4132ea396a25) in this PR that shouldn't be here tho 🤔

File: app/src/main/res/values/strings.xml (hunk 2 above)
Comment (kaungkhantsoe) [reply app/src/main/res/values/strings.xml]: Yeah it is weird. AS doesn't show conflict after rebasing but GitHub was having conflict. So I pushed the code and it turned out like this. 😞

**Review by ryan-conway — APPROVED** [review]
🙏🏻

**Review by AVI5HEK — APPROVED** [review]

**Pull request merged**